addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -1417,6 +1417,12 @@ def test_routing_loop_alias(self):
         self.env['ir.config_parameter'].sudo().set_param('mail.gateway.loop.minutes', 30)
         self.env['ir.config_parameter'].sudo().set_param('mail.gateway.loop.threshold', 5)
 
+        self.env['mail.gateway.whitelist'].create([
+            {'email': "Bob@EXAMPLE.com"},
+            {'email': "alice@EXAMPLE.com"},
+            {'email': "eve@EXAMPLE.com"},
```

Comment:
Just to have a test no normalized email, we could have '"Eve From Example" <eve@example.COM>'